scripts/test-harness-contracts.ps1

```
@@ -2605,6 +2605,53 @@ try {
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' 'HIGHEST effort is reserved for the last filter' 'task-cycle §8: the platform maximum is reserved for the final filter'
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' 'escalation after a review rejection' 'task-cycle §8: earlier maximum effort requires a rejection first'
+Check-NotMatch '.claude/skills/pelizzai-execute/references/task-cycle.md' 'stays at the highest level the user''s platform offers' 'task-cycle §8: the universal maximum-effort mandate is gone'
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' 'raise the reasoning effort of the NEXT\s+fix and re-review dispatches' 'task-cycle §5: escalation by failure raises the next round, not the first pass'
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' 'LARGER MODEL remains a ratifiable recommendation' 'task-cycle §5: a larger model still requires ratification'
+# The final filter KEEPS the maximum — the proportionality must not erode the last line of defense.
+Check-Match '.claude/skills/pelizzai-review/SKILL.md' 'highest effort the platform allows' 'review: the final branch review keeps the platform maximum'
+Check-Match 'CLAUDE.md' 'reserve the platform''s highest effort for the final review' 'CLAUDE.md carries the proportional effort doctrine'
+Check-Match '.claude/skills/pelizzai-audit/assets/contract.md' 'reserve the platform''s highest effort for the final review' 'the consumer contract asset carries the same doctrine'
+
+# -- (d) Phase timing exists, travels in the existing record, and creates nothing --
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' '## 9\. Phase timing' 'task-cycle has the phase-timing section'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)